sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statements.scala

```diff
@@ -66,7 +66,6 @@ case class CreateTableStatement(
     properties: Map[String, String],
     provider: String,
     options: Map[String, String],
-    location: Option[String],
```

Comment:
After more thoughts, I think we should keep `location` in the logical plan field not table properties after the parser, to be consistent with provider and comment.

Reply:
ok